packages/core/src/animation/AnimationClipCurveData.ts

```diff
@@ -0,0 +1,22 @@
+import { InterpolableValue } from "./KeyFrame";
+import { Entity } from "./../Entity";
+import { Component } from "../Component";
+import { AnimationCurve } from "./AnimationCurve";
+
+/**
+ * Associate AnimationCurve and the Entity
```

Comment:
I think this interface is internal.
add @internal here, otherwise it will be published in the document

Reply:
done